# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/PBackgroundIDBCursor.ipdl
# Commit: 6818224b9b11
# Full Hash: 6818224b9b119944f818554e4071a566bca4ed63
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-11-05 21:49:23
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Add support for preloading key-only cursors. r=ttung,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44009
# ==============================================================================

diff -r f1be81af7d6b -r 6818224b9b11 dom/indexedDB/PBackgroundIDBCursor.ipdl
--- a/dom/indexedDB/PBackgroundIDBCursor.ipdl	Tue Nov 05 12:45:48 2019 +0000
+++ b/dom/indexedDB/PBackgroundIDBCursor.ipdl	Tue Nov 05 12:46:16 2019 +0000
@@ -79,9 +79,9 @@
   void_t;
   nsresult;
   ObjectStoreCursorResponse[];
-  ObjectStoreKeyCursorResponse;
+  ObjectStoreKeyCursorResponse[];
   IndexCursorResponse[];
-  IndexKeyCursorResponse;
+  IndexKeyCursorResponse[];
 };
 
 protocol PBackgroundIDBCursor
